operations/host_spawn.go

```diff
@@ -89,29 +95,42 @@ func hostCreate() cli.Command {
 			client := conf.setupRestCommunicator(ctx)
 			defer client.Close()
 
-			var script string
+			spawnRequest := &restModel.HostRequestOptions{}
+			tags := []host.Tag 
+			if file != "" {
+				err = utility.ReadYAMLFile(file, &spawnRequest)
+				if err != nil {
+					return errors.Wrapf(err, "problem reading from file '%s'", file)
+				}
```

Comment:
I realize it's messy that host.create uses one interface and spawn hosts use another. Since the point of this file is to allow `host.create` scripts to be passed into the CLI, I think it's okay to restrict this to only the subset of host.create parameters that we want to support, and since tags isn't a host.create parameter we don't need to handle it in the file.

Reply:
That is a good point. However, since it's already working, I see no reason to remove the functionality since someone might have a different use case and find it useful. Additionally, not supporting it will end up requiring more documentation and potentially open more questions from users down the line. Thoughts?

Reply:
Okay sounds good, as long as the documentation is clear!